src/pybind/mgr/email/module.py

```diff
@@ -0,0 +1,55 @@
+
+"""
+An email module
+
+See doc/mgr/hello.rst for more info.
+"""
+
+from mgr_module import MgrModule
+import smtplib
+
+class Email(MgrModule):
+    COMMANDS = [
+        {
+            "cmd": "Email "
+                   "name=port,type=CephInt,req=false "
+                   "name=host,type=CephString,req=false "
+                   "name=sender_id,type=CephString,req=true "
+                   "name=target_id,type=CephString,req=true "
+                   "name=msg,type=CephString,req=true",
+            "desc": "Sends email",
+            "perm": "rw"
+        },
+    ]
+
+    def handle_command(self, inbuf, cmd):
+        self.log.info("Email_info")
+        self.log.debug("Email_debug")
+        self.log.error("Email_error")
+
+        status_code = 0
+        output_buffer = "Output buffer is for data results"
+        output_string = "Output string is for informative text"
+
+        sender = cmd['sender_id']
+        receivers = cmd['target_id']
+
+        email_message = cmd['msg']
```

Comment:
Let's try taking the result of self.get('health') and including that content into the message.  Either dump it out (e.g., '%s' % self.get('health')), or try to iterate over it to generate a more human-readable email message that looks something like what you get from 'ceph health detail'.